python/idsse_common/test/test_path_builder.py

```diff
@@ -171,11 +171,22 @@ def test_get_valid_returns_none_if_args_empty():
 def test_get_valid_from_time_args_calculates_based_on_lead(path_builder: PathBuilder):
     parsed_dict = path_builder.parse_path(EXAMPLE_FULL_PATH)
     result_valid: datetime = PathBuilder.get_valid_from_time_args(parsed_args=parsed_dict)
-
     assert result_valid == EXAMPLE_VALID
 
 
 def test_get_lead_from_time_args(path_builder: PathBuilder):
     parsed_dict = path_builder.parse_path(EXAMPLE_FULL_PATH)
     lead_result: timedelta = PathBuilder.get_lead_from_time_args(parsed_dict)
     assert lead_result.seconds == EXAMPLE_LEAD.minute * 60
+
+
+def test_calculate_issue_from_valid_and_lead():
+    parsed_dict = {}
+    parsed_dict['valid.year'] = 1970
+    parsed_dict['valid.month'] = 10
+    parsed_dict['valid.day'] = 3
+    parsed_dict['valid.hour'] = 14
+    parsed_dict['lead.hour'] = 2
```

Comment:
```suggestion
    parsed_dict = {
        'valid.year': 1970,
        'valid.month': 10,
        'valid.day': 3,
        'valid.hour': 14,
        'lead.hour': 2
    }
```
I prefer to define my dictionaries like this, but don't know if there is any real reason (time, storage, etc).

Reply:
Agreed, and it seems more readable too.